test/extended/oauth/server_headers.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/openshift/origin/test/extended/util/oauthserver"
 )
 
-var _ = g.Describe("[sig-auth][Feature:OAuthServer] [Headers][apigroup:route.openshift.io]", func() {
+var _ = g.Describe("[sig-auth][Feature:OAuthServer] [Headers][apigroup:route.openshift.io][apigroup:config.openshift.io]", func() {
```

Comment:
I see. `deployOAuthServer` -> `DeployOAuthServer` -> `getImage` -> `determineImageFromRelease` -> `DetermineImageFromRelease`

Also `deployOAuthServer` invokes `oauthserver.DeployOAuthServer` which invokes `createOAuthClient` which uses `OauthV1`. So at least `[apigroup:oauth.openshift.io]` is needed as well.

Reply:
Nice catch, this one slipped.